backend/lib/endpoints/posts.js

```diff
@@ -52,18 +52,33 @@ async function routes(app) {
           limit,
           objective,
           skip,
+          postMode,
           includeMeta,
         },
       } = req;
+
       const queryFilters = filter ? JSON.parse(decodeURIComponent(filter)) : {};
       // Base filters - expiration and visibility
       /* eslint-disable sort-keys */
-      const filters = [
+      let filters = [
         {
           $or: [{ expireAt: null }, { expireAt: { $gt: new Date() } }],
           status: { $ne: "removed" },
         },
       ];
+      // "IA" stands for InActive or Archived
+      //In future we will also have "D" for Drafts
+      if (postMode === "IA") {
```

Comment:
I think it's better to use enum for this input, e.g. POST_MODE, [this one](https://github.com/FightPandemics/FightPandemics/blob/7190df6f87b09bcd7fa26e4d1d348ca095605c46/backend/lib/models/Post.js#L4) is a reference

Reply:
currently this filter is different from above one only for the Date part, maybe use a conditional statement to simplify it, like `postMode === 'IA' ? $lt ... : $gt ...

Reply:
> currently this filter is different from above one only for the Date part, maybe use a conditional statement to simplify it, like `postMode === 'IA' ? $lt ... : $gt ...

This is Done too.